src/pages/index.js

```diff
@@ -129,13 +129,14 @@ updateProfileButton.addEventListener("click", () => {
 
 updateAvatarButton.addEventListener("click", () => {
   avatarPopup.open();
+  toggleButtonState.open() //I'm lost on this, could I have an example?
```

Comment:
Please, delete it. You don't have such an instance


You need to call `updateAvatarFormValidator.toggleButtonState` to disable the avatar submit button


```js

 updateAvatarButton.addEventListener("click", () => {
  avatarPopup.open();
  updateAvatarFormValidator.toggleButtonState()
});
```